src/pages/user/User.tsx

```diff
@@ -84,15 +85,20 @@ const User: React.FC = () => {
   return (
     <StyledUserPage>
       <StyledSecondaryUserInfo>
-        <UserCard headingLabel={t('common:picture')} />
-        <UserCard headingLabel={t('heading.contact_info')} />
+        <Card>
+          <Heading>{t('common:picture')} </Heading>
+        </Card>
+        <Card>
+          <Heading>{t('heading.contact_info')} </Heading>
```

Comment:
håhå, ekstra space. samme for Heading over

Reply:
og linje 122

Reply:
knegg